pyvo/registry/rtcons.py

```diff
@@ -284,14 +318,15 @@ def _get_or_condition(self, service):
         return super().get_search_condition(service)
 
 
-class Author(Constraint):
+class Author(SubqueriedConstraint):
```

Comment:
sphinx is failing as in the API docs it tries to link to the baseclass, but the new class hasn't been added to the public API and therefore there is no API docs available for it.

I'll add a commit that resolves this issue, take it or leave it (I mean we can work around not exposing the class in the public API if necessary).

FWIW, `Constraint` is in the public API, so I don't think it's a big problem to add the new one, too.